server/src/common/dto/save-application.dto.ts

```diff
@@ -1,10 +1,9 @@
-import { ReviewStatuses } from '../enums';
-
 export declare class SaveApplicationDto {
```

Comment:
https://docs.nestjs.com/controllers#request-payloads - Rename this to be an interface